x-pack/plugin/ql/test-fixtures/src/main/java/org/elasticsearch/xpack/ql/TestUtils.java

```diff
@@ -67,15 +73,19 @@
 import static org.elasticsearch.test.ESTestCase.randomBoolean;
 import static org.elasticsearch.test.ESTestCase.randomFrom;
 import static org.elasticsearch.test.ESTestCase.randomZone;
+import static org.elasticsearch.xpack.ql.TestUtils.StringContainsRegex.containsRegex;
 import static org.elasticsearch.xpack.ql.tree.Source.EMPTY;
+import static org.hamcrest.Matchers.containsString;
 import static org.junit.Assert.assertEquals;
 
 public final class TestUtils {
 
     public static final ZoneId UTC = ZoneId.of("Z");
-
     public static final Configuration TEST_CFG = new Configuration(UTC, null, null);
 
+    private static final String MATCHER_TYPE_CONTAINS = "CONTAINS";
```

Comment:
Any reason why contains need to be specified? Since it's an either/or option - either there's nothing specified (contains) or something else such as regex - specifying contains is redundant.

Reply:
I kept it, just because it could help when you have multiple assertions of mixed regex and simple contains and could make sense to have something like:
```
contains <match1>
regex <match2>
regex <match3>
contains <match4>
...
```
Since it doesn't complicate the code (it's only a couple of extra lines), I thought it would be nice to have it as an option. If you insist  though I can remove it and keep only the implicit version.